app/src/utils/utils.ts

```diff
@@ -13,10 +13,19 @@ export function formatAddress(address: string) {
   return `${address.slice(0, 6)}…${address.slice(38)}`;
 }
 
+// Is the given val defined?
 export function isDefined(val: unknown) {
   return !!val;
 }
 
+// Accepts number or string and rounds it to the given number of decimals
+export function roundNumber(number: string | number, decimals: number) {
+  const mul = 10 ** decimals;
+  const num = typeof number == 'number' ? number : parseFloat(number);
+
+  return Math.round(num * mul) / mul;
+}
```

Comment:
I have a similar function in my branch like this, but it also adds commas for large numbers. Feel free to use/combine these two if you'd like (`commify` comes from ethers in `src/utils/ethers`). I named it `formatNumber` to avoid naming confusion of rounding numbers with the concept of a "grant round" 🤷

```typescript
// Formats a number for display to the user, by rounding to the specified number of decimals and adding commas
export function formatNumber(value: BigNumberish, decimals: number): string {
  // `BigNumber.from()` can't take decimal inputs
  value = typeof number === 'number' || typeof number === 'string' ? Number(value) : BigNumber.from(value);
  return commify(parseFloat(value.toFixed(decimals)));
}
```

Reply:
I've copied it from here but altered it a little bit:

```ts
// Formats a number for display to the user, by rounding to the specified number of decimals and adding commas
export function formatNumber(value: BigNumberish, decimals: number): string {
  // `BigNumber.from()` can't take decimal inputs
  value = typeof value === 'number' || typeof value === 'string' ? Number(value) : BigNumber.from(value);
  return commify(parseFloat(String(value)).toFixed(decimals));
}
```

Reply:
LGTM! I'll rebase against your latest branch and use this version also